api/src/main/java/org/pmiops/workbench/profile/ProfileService.java

```diff
@@ -432,10 +492,48 @@ public void validateProfile(@Nonnull Profile updatedProfile, @Nullable Profile p
    * @throws BadRequestException
    */
   public void validateNewProfile(Profile profile) throws BadRequestException {
-    validateProfile(profile, null);
+    // unused, but this is the correct syntax for a "Diff" of a new object
+    final Diff dummyDiff = javers.compare(null, profile);
+    validateProfileForCorrectness(profile, true, dummyDiff);
   }
 
   public List<Profile> listAllProfiles() {
     return userService.getAllUsers().stream().map(this::getProfile).collect(Collectors.toList());
   }
+
+  /**
+   * Updates the user metadata referenced by the fields of EditUserInformationRequest.
+   *
+   * @param request the fields to update. Fields left null here will not be updated. Contact Email
+   *     and Verified Institutional Affiliation updates will trigger a check for affiliation
+   *     validation.
+   * @return the Profile of the user, after updates
+   */
+  public Profile editUserInformation(EditUserInformationRequest request) {
+    final DbUser dbUser = userService.getByUsernameOrThrow(request.getUsername());
+    final Profile originalProfile = getProfile(dbUser);
```

Comment:
I'd bundle these two calls into a `ProfileService#getByUsername(String username)` if you don't need the DbUser.

Reply:
I do need the DbUser